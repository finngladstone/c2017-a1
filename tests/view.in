view
place A1
view
who
place A2
place A3
place A4
view
place A19
place S10
place S4
view
resign
