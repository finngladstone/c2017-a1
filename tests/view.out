J10,.................................................
M3,..........................................xxxxxxx
W
M12,.................................................
E12,.................................................
Black wins!
A1A2A3A4A19S10S4
Thank you for playing!
